lib/api/WebApi.js

```diff
@@ -87,6 +87,16 @@ async function getLocationByCentreline(feature) {
   return location;
 }
 
+async function getLocationsByCorridor(features) {
+  if (features.length === 0) {
+    return [];
+  }
+  const s1 = CompositeId.encode(features);
+  const data = { s1 };
+  const options = { data };
+  return apiClient.fetch('/locations/byCorridor', options);
```

Comment:
now that we have response schemas on `LocationController`, can we enforce them in the client as well just like for other endpoints?